example/getting_started/example1/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2020, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
+project(powsybl_iidm4cpp_getting_started_1)
+
+find_package(Boost 1.56 REQUIRED COMPONENTS date_time filesystem program_options system unit_test_framework)
+find_package(LibXml2 REQUIRED)
+find_package(LibIIDM REQUIRED)
```

Comment:
Do you really need to find all these libraries? Make this file as simpler you can to avoid bad copy/paste for our users

Reply:
The find package are mandatory for Boost and Libxml2. They became mandatory the moment you use targets. But the error messages if you don't put the two first lines are pretty simple to interpret:
```
Target "iidm4cpp_reader" links to target "LibXml2::LibXml2" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
```